<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2dxcustomize</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/assets.qrc">
    <normaloff>:/graphics/icon.ico</normaloff>
    <normalon>:/graphics/icon.ico</normalon>
    <disabledoff>:/graphics/icon.ico</disabledoff>
    <disabledon>:/graphics/icon.ico</disabledon>
    <activeoff>:/graphics/icon.ico</activeoff>
    <activeon>:/graphics/icon.ico</activeon>
    <selectedoff>:/graphics/icon.ico</selectedoff>
    <selectedon>:/graphics/icon.ico</selectedon>:/graphics/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>461</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="burst_label">
       <property name="text">
        <string>Burst</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="burst_items"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="turntable_label">
       <property name="text">
        <string>Turntable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="turntable_items"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lane_cover_label">
       <property name="text">
        <string>Lane cover</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="lane_cover_items"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="judge_font_label">
       <property name="text">
        <string>Judge font</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="judge_font_items"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="note_skin_label">
       <property name="text">
        <string>Note skin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="note_skin_items"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="full_combo_effect_label">
       <property name="text">
        <string>Full combo</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="full_combo_effect_items"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lane_beam_label">
       <property name="text">
        <string>Lane beam</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="lane_beam_items"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="github_link">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>github.com/em_skye96</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="banner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>87</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res/assets.qrc"/>
 </resources>
 <connections/>
</ui>
